apps/common-app/src/apps/reanimated/examples/ShadowNodesCloningExample.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import Animated, { useSharedValue, withTiming } from 'react-native-reanimated';
+
+function AnimatedView() {
+  const opacitySv = useSharedValue(0);
+  const scaleSv = useSharedValue(0);
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      opacitySv.value = withTiming(Math.random(), { duration: 300 });
+      scaleSv.value = withTiming(Math.random() * 2, { duration: 300 });
+      setCounter((prevCounter) => prevCounter + 1);
+    }, 1000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, [opacitySv, scaleSv, setCounter]);
+
+  const style = [
+    styles.animatedView,
+    {
+      opacity: opacitySv,
+      transform: [{ scale: scaleSv }],
+    },
+  ];
+
+  return (
+    <Animated.View collapsable={false} style={style}>
+      <Text>{counter}</Text>
+    </Animated.View>
+  );
+}
+
+function RecursiveView(props: { depth: number; noExtraChildren: number }) {
```

Comment:
The name `noExtraChildren` suggests that the props is a boolean but it's a number, can we rename it to something else like `extraChildrenCount`?